IOSDeviceLib/SocketHelper.h

```diff
@@ -28,9 +29,22 @@ struct LengthEncodedMessage {
 	}
 };
 
+struct Utf16Message {
+	unsigned char *message;
+	long long length;
+
+	~Utf16Message()
+	{
+		free(message);
```

Comment:
As you're allocating memory for `message` with the `new` operator it seems semantically more correct to me to use `delete[]` here instead of `free`